tests/Unit/CompanyTest.php

```diff
@@ -0,0 +1,382 @@
+     */
+    public function testItHasRules()
+    {
+        $company = $this->company(self::TEST_COMPANY);
+
+        $this->assertIsString($company->getRules());
+    }
+
+    /**
+     * @throws \Http\Client\Exception
+     * @throws \Phpfastcache\Exceptions\PhpfastcacheInvalidArgumentException
+     * @throws \TruckersMP\Exceptions\PageNotFoundException
+     * @throws \TruckersMP\Exceptions\RequestException
+     */
+    public function testItHasRequirements()
+    {
+        $company = $this->company(self::TEST_COMPANY);
+
+        $this->assertIsString($company->getRequirements());
+    }
+
+    /**
+     * @throws \Http\Client\Exception
+     * @throws \Phpfastcache\Exceptions\PhpfastcacheInvalidArgumentException
+     * @throws \TruckersMP\Exceptions\PageNotFoundException
+     * @throws \TruckersMP\Exceptions\RequestException
+     */
+    public function testItHasAWebsite()
+    {
+        $company = $this->company(self::TEST_COMPANY);
+
+        $this->assertIsString($company->getWebsite());
+    }
+
+    /**
+     * @throws \Http\Client\Exception
+     * @throws \Phpfastcache\Exceptions\PhpfastcacheInvalidArgumentException
+     * @throws \TruckersMP\Exceptions\PageNotFoundException
+     * @throws \TruckersMP\Exceptions\RequestException
+     */
+    public function testItHasSocialInformation()
+    {
+        $company = $this->company(self::TEST_COMPANY);
+
+        $this->assertInstanceOf(Social::class, $company->getSocial());
+
+        // Twitter
+        if ($company->getSocial()->getTwitter() !== null) {
+            $this->assertIsString($company->getSocial()->getTwitter());
+        } else {
+            $this->assertNull($company->getSocial()->getTwitter());
+        }
+
+        // Facebook
+        if ($company->getSocial()->getFacebook() !== null) {
+            $this->assertIsString($company->getSocial()->getFacebook());
+        } else {
+            $this->assertNull($company->getSocial()->getFacebook());
+        }
+
+        // Twitch
+        if ($company->getSocial()->getTwitch() !== null) {
+            $this->assertIsString($company->getSocial()->getTwitch());
+        } else {
+            $this->assertNull($company->getSocial()->getTwitch());
+        }
+
+        // PlaysTV
+        if ($company->getSocial()->getPlays() !== null) {
+            $this->assertIsString($company->getSocial()->getPlays());
+        } else {
+            $this->assertNull($company->getSocial()->getPlays());
+        }
+
+        // Discord
+        if ($company->getSocial()->getDiscord() !== null) {
+            $this->assertIsString($company->getSocial()->getDiscord());
+        } else {
+            $this->assertNull($company->getSocial()->getDiscord());
+        }
+
+        // YouTube
+        if ($company->getSocial()->getYouTube() !== null) {
+            $this->assertIsString($company->getSocial()->getYouTube());
+        } else {
+            $this->assertNull($company->getSocial()->getYouTube());
```

Comment:
Looking at the tests now, this seems to be useless as this assertation happens only when the social information is `null`.

Reply:
Gone ahead and removed asserting if `null`, also added a check to make sure `getSocial()` returns an instance of the `Social` model